README.md

```diff
@@ -1,2 +1,99 @@
 # hbv_nano
-HBV Nanopore
+
+hbv_nano is a pipeline for analysing whole genome Nanopore long-read data for resistance typing of hepatitis B viruses (HBV).
+
+## Requirements
+To run the pipleline `Mamba` and `Apptainer` have to be installed.
+
+## Installation
+### 1. Clone the repository and change into it.
+Clone the repository
+```
+git clone -b hbv_nano_daniel https://github.com/ClinicalGenomicsGBG/hbv_nano.git
+```
+and change the location to it.
+```
+cd /hbv_nano
+```
+
+### 2. Create a mamba base enironment to run the pipeline:
+```
+mamba env create -f workflow/scripts/environment.yaml
+```
+If you want to create the environment in a specific location add the flag `-p <path>`
+```
+mamba env create -f workflow/scripts/environment.yaml -p <path>
+```
+
+### 3. Build Apptainer images for the different software needed to run the pipeline.
+Make sure that you are still in the location ``.../hbv_nano``. To build the images run the script `build_images.sh`:
+```
+./build_images.sh
```

Comment:
Might be good to put this as `bash ./build_images.sh` to avoid confusion/issues with shebang/permissions.